tests/integration/targets/aci_l3out_floating_svi/tasks/main.yml

```diff
@@ -0,0 +1,220 @@
+# Author: Shreyas Srish (@shrsr)
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# SET VARS
+- name: Set vars
+  set_fact:
+    aci_info: &aci_info
+      host: '{{ aci_hostname }}'
+      username: '{{ aci_username }}'
+      password: '{{ aci_password }}'
+      validate_certs: '{{ aci_validate_certs | default(false) }}'
+      use_ssl: '{{ aci_use_ssl | default(true) }}'
+      use_proxy: '{{ aci_use_proxy | default(true) }}'
+      output_level: '{{ aci_output_level | default("info") }}'
+
+# CLEAN ENVIRONMENT
+- name: Remove test tenant before we kickoff
+  cisco.aci.aci_tenant: &tenant_absent
+    <<: *aci_info
+    tenant: ansible_test
+    state: absent
+
+- name: Verify Cloud and Non-Cloud Sites in use.
+  include_tasks: ../../../../../../integration/targets/aci_cloud_provider/tasks/main.yml
+
+- name: Execute tasks only for non-cloud sites
+  when: query_cloud.current == []  # This condition will execute only non-cloud sites
+  block:  # block specifies execution of tasks within, based on conditions
+  # SETUP ENVIRONMENT
+  - name: Create domain
+    cisco.aci.aci_domain: &domain_present
+      <<: *aci_info
+      domain: l3outintftest
+      domain_type: l3dom
+      state: present
+
+  - name: Create tenant
+    cisco.aci.aci_tenant: &tenant_present
+      <<: *tenant_absent
+      state: present
+
+  - name: Configure VRF
+    cisco.aci.aci_vrf: &vrf_present
+      <<: *tenant_present
+      vrf: l3outintftest
+
+  - name: Create L3Out
+    cisco.aci.aci_l3out:
+      <<: *vrf_present
+      l3out: l3outintftest
+      domain: l3outintftest
+      route_control: export
+
+  - name: Create node profile
+    cisco.aci.aci_l3out_logical_node_profile: &np_present
+      <<: *tenant_present
+      l3out: l3outintftest
+      node_profile: nodes
+
+  - name: Add interface profile
+    cisco.aci.aci_l3out_logical_interface_profile: &intf_present
+      <<: *np_present
+      interface_profile: Floating
+
+  - name: Create a floating svi in check mode
+    cisco.aci.aci_l3out_floating_svi: &floating_svi
+      <<: *intf_present
+      pod_id: 1
+      node_id: 201
+      encap: vlan-1
+      address: 23.45.67.90/24
+      external_bridge_group_profile: bridge1
+      state: present
+    check_mode: true
+    register: add_floating_cm
+    when: version.current.0.topSystem.attributes.version is version('5', '>=')
```

Comment:
I am confused now with the tests of aci_l3out_floating_svi

To my understanding aci_l3out_floating_svi (l3extVirtualLIfP) in configurable on <5 only the external_bridge_group_profile (l3extBdProfileCont) is a unsupported class on <5. 

When I execute the create with "external_bridge_group_profile" commented out, see below:

```
- name: Create a floating svi in check mode
    cisco.aci.aci_l3out_floating_svi: &floating_svi
      <<: *intf_present
      pod_id: 1
      node_id: 201
      encap: vlan-1
      address: 23.45.67.90/24
      # external_bridge_group_profile: bridge1
      state: present
    register: add_floating_cm
```

The result is:
```
changed: [cn-dmz-apic-m1-02-v42] => {
    "changed": true,
    "current": [
        {
            "l3extVirtualLIfP": {
                "attributes": {
                    "addr": "23.45.67.90/24",
                    "annotation": "orchestrator:ansible",
                    "autostate": "disabled",
                    "descr": "",
                    "dn": "uni/tn-ansible_test/out-l3outintftest/lnodep-nodes/lifp-Floating/vlifp-[topology/pod-1/node-201]-[vlan-1]",
                    "encap": "vlan-1",
                    "encapScope": "local",
                    "ifInstT": "ext-svi",
                    "ipv6Dad": "enabled",
                    "llAddr": "::",
                    "mac": "00:22:BD:F8:19:FF",
                    "mode": "regular",
                    "mtu": "inherit",
                    "name": "",
                    "nodeDn": "topology/pod-1/node-201",
                    "targetDscp": "unspecified"
                }
            }
        }
    ],
```

Why are we not testing aci_l3out_floating_svi on version below 5 without the external_bridge_group_profile?

From https://www.cisco.com/c/en/us/td/docs/dcn/aci/apic/all/floating-l3outs/simplify-outside-network-connections-using-floating-l3outs/m-configuring-floating-l3outs.html I understand that Floating L3Out with VMM domain requires Cisco Application Policy Infrastructure Controller (APIC) Release 4.2(1) or later. 

If this is not a scenario we provide in the module should we then restrict the entire module for pre 5.0 and document this in the module description? Instead of providing only `This is only supported in APIC v5.0 and above.` under the external_bridge_group_profile

Reply:
Testing it on 4.2 below